packages/@sanity/form-builder/src/inputs/BlockEditor/nodes/styles/ListItem.css

```diff
@@ -70,49 +70,42 @@
   composes: root;
 
   @nest & > .item:before {
-    content: '\25CF '; /* BLACK CIRCLE */
+    content: '  \25CF'; /* BLACK CIRCLE */
+    font-size: 0.4375rem; /* 7px */
+    line-height: 1.5rem; /* Same as body text */
```

Comment:
Shouldn't this be `em` or `inherit` since people can override line height?

Reply:
Afaik, there's no way of using `em` or inheriting the same the line height in this case.

Reply:
I ment the font-size should be relative (`em`). 
Then the line-height also can be relative. It is possible to override the font-size of the editor.

Reply:
Unitless line-height is relative to the font size afaik.